packages/vite/src/node/packages.ts

```diff
@@ -174,8 +174,15 @@ export function findNearestMainPackageData(
   )
 }
 
+function stripBOM(content: string) {
+  if (content.charCodeAt(0) === 0xfeff) {
+    content = content.slice(1)
+  }
+  return content
+}
```

Comment:
This function already exists at
https://github.com/vitejs/vite/blob/2b4f115129fb3fbd730a92078acb724f8527b7f7/packages/vite/src/node/utils.ts#L1323
Would you use that instead?

Reply:
yes, I would